test/contrib/isotp_soft_socket.uts

```diff
@@ -1,19 +1,29 @@
 % Regression tests for ISOTPSoftSocket
-~ automotive_comm disabled
```

Comment:
Can you make sure this isn't run in `./run_tests` ?

Reply:
I will leave the automotive_comm. Thanks for reminding me.